corehq/apps/users/analytics.py

```diff
@@ -71,8 +71,8 @@ def get_search_users_in_domain_es_query(domain, search_string, limit, offset):
     default_search_fields = ["base_username", "last_name", "first_name"]
     return (UserES()
             .domain(domain)
-            .OR(users.metadata('login_as_user', search_string),
-                queries.search_string_query(search_string, default_search_fields))
+            .filter(filters.OR(users.metadata('login_as_user', search_string),
```

Comment:
what's with the change from `.OR()` to `.filter(filters.OR())`?  The former is syntactic sugar for the latter - they should produce identical queries.

Reply:
Ha, please ignore that was just my lame attempt at trying to trick the test case :P